util/execdetails/execdetails.go

```diff
@@ -86,8 +86,10 @@ const (
 	CommitBackoffTimeStr = "Commit_backoff_time"
 	// BackoffTypesStr means the backoff type.
 	BackoffTypesStr = "Backoff_types"
-	// SlowestCommitRPCDetailStr means the details of the slowest RPC during the transaction commit process.
-	SlowestCommitRPCDetailStr = "Slowest_commit_rpc_detail"
+	// SlowestPrewriteRPCDetailStr means the details of the slowest RPC during the transaction 2pc prewrite process.
+	SlowestPrewriteRPCDetailStr = "Slowest_prewrite_rpc_detail"
+	// CommitPrimaryRPCDetailStr means the details of the slowest RPC during the transaction 2pc commit process.
```

Comment:
Only commit primary?

Reply:
For the query latency calculation of 2pc committed transactions, the primary key commit is enough.

Reply:
I mean the comment message is slightly inaccurate.